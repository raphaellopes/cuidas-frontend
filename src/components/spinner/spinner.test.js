// vendors
import React from 'react';
import renderer from 'react-test-renderer';

// locals
import Spinner from './index';

const component = () => (renderer.create(
  <Spinner />,
));

describe('<Spinner />', () => {
  test('Should render the component', () => {
    const tree = component().toJSON();
    expect(tree).toMatchSnapshot();
  });
});
