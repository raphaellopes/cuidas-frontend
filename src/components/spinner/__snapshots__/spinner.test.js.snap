// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner /> Should render the component 1`] = `
<i
  className="icon fa fa-asterisk fa-spin sc-bdVaJa jToUlD"
/>
`;
